source/_integrations/squeezebox.markdown

```diff
@@ -21,7 +21,7 @@ ha_integration_type: integration
 
 The Squeezebox integration allows you to control music players from the [Lyrion Music Server](https://lyrion.org/) (LMS) ecosystem.  Lyrion Music Server was formerly known as [Logitech Media Server](https://en.wikipedia.org/wiki/Squeezebox_%28network_music_player%29).
 
-This integration provides both media players connected to the server and supporting binary sensors for the server status.
+This integration connects to an existing LMS server and provides both media players and sensors for monitoring server status.
```

Comment:
```suggestion
This integration connects to an existing <abbr title="Lyrion Music Server">LMS</abbr> server and provides both media players and sensors for monitoring server status.
```
we have these tooltips for abbreviations
![image](https://github.com/user-attachments/assets/d6081697-7d8c-4367-be98-9ff3195d106d)